tests/index.test.js

```diff
@@ -0,0 +1,387 @@
+describe('sumIfEven', () => {
+    it('should return sum (odd number)', () => {
+        const first = 5;
+        const second = 5;
+        const expected = 10;
+
+        const actual = sumIfEven(first, second);
+
+        assert.deepEqual(actual, expected);
+    });
+    
+    it('should return product (even number)', () => {
+        const first = 2;
+        const second = 5;
+        const expected = 10;
+
+        const actual = sumIfEven(first, second);
+
+        assert.deepEqual(actual, expected);
+    });
+});
+
+describe('addPositives', () => {
+    it(' should return result ', () => {
+        const first = 5;
+        const second = 5;
+        const expected = 10;
+
+        const actual = sumIfEven(first, second);
```

Comment:
maybe in test for `addPositives` we should call `addPositives` ?
Also please add cases if one of numbers is negative, two of three numbers are negative, all three numbers are negative, all of three numbers are zeros, etc.